catalyst_voices/Earthfile

```diff
@@ -71,13 +87,13 @@ build:
     WORKDIR /frontend/build
     SAVE ARTIFACT web /web AS LOCAL web
 
-test-unit:
-    FROM +build
-    WORKDIR /frontend
-    TRY
-        RUN flutter test --reporter expanded . --machine | tojunit --output flutter.junit-report.xml
-    FINALLY
-        SAVE ARTIFACT flutter.junit-report.xml AS LOCAL flutter-unit-tests.junit-report.xml
+test-report:
```

Comment:
Why changing the name of the target? I would keep it as test-unit since its main function is to run unit/widget tests?

Reply:
I was thinking about what does this target do and came to a conclusion that primarily you would run it to generate the junit report or coverage. I'll revert the old name 👍